### PR TITLE
CLC-6398, Course Capture widget: trimming, padding and verbiage

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-6398

See Jira above for expected renderings.

### CODE REVIEW

File: src/assets/stylesheets/_webcast.scss [hunk 1]
```diff
@@ -10,9 +10,22 @@
...
+    padding-top: 15px;
+  }
   .cc-widget-webcast-content {
     max-width: 480px;
   }
+  .cc-webcast-table {
+    table {
+      tr {
+        td, th {
+          padding: 5px 5px 5px 5px;
```
Comment (paulkerschen) [inline src/assets/stylesheets/_webcast.scss]: @johncrossman Will the CSS linter ding you for not just including `5px` once?

Comment (johncrossman) [inline src/assets/stylesheets/_webcast.scss]: @paulkerschen We'll find out very soon. Nonetheless, I'll amend.

File: src/assets/templates/widgets/webcast.html [hunk 2]
```diff
@@ -73,41 +63,46 @@ <h1 class="cc-visuallyhidden" data-ng-if="isEmbedded">Course Capture Sign-up</h1
...
-              <i class="fa fa-exclamation-triangle cc-icon-gold"></i>
-              <strong>Alert: </strong>
-              You must log in to YouTube with your bConnected account to watch the videos below. For more information,
-              go to our <a href="https://berkeley.service-now.com/kb_view.do?sysparm_article=KB0011469">help page</a>.
-            </div>
-            <div class="cc-table" data-ng-repeat="medium in media">
-              <h3 data-ng-bind-template="{{medium.deptName + ' ' + medium.catalogId + ' ' + medium.instructionFormat + ' ' + medium.sectionNumber}}"
-                data-ng-if="media.length > 1"></h3>
-              <table>
+          <div data-ng-if="media.length > 0">
```
Comment (paulkerschen) [inline src/assets/templates/widgets/webcast.html]: @johncrossman I think house style is to just rely on Angular short-circuiting and abbreviate to `data-ng-if="media.length"`?

File: src/assets/templates/widgets/webcast.html [hunk 3]
```diff
@@ -73,41 +63,46 @@ <h1 class="cc-visuallyhidden" data-ng-if="isEmbedded">Course Capture Sign-up</h1
...
+              <h3 data-ng-class="{'cc-widget-section-header': !$first}" data-ng-bind-template="{{section.deptName + ' ' + section.catalogId + ' ' + section.instructionFormat + ' ' + section.sectionNumber}}" data-ng-if="media.length > 1"></h3>
+
+              <div class="cc-widget-webcast-alert" data-ng-if="section.videos.length > 0">
+                <i class="fa fa-exclamation-triangle cc-icon-gold"></i>
+                <strong>Alert: </strong>
+                Log in to YouTube with your bConnected account to watch the videos below. More information is at
+                our <a href="https://berkeley.service-now.com/kb_view.do?sysparm_article=KB0011469">help page</a>.
+              </div>
+
+              <div data-ng-if="(section.videos.length === 0) && (section.audio.length === 0)">
```
Comment (paulkerschen) [inline src/assets/templates/widgets/webcast.html]: @johncrossman Likewise here: `data-ng-if="!section.videos.length && !section.audio.length"`?

**Comment by johncrossman** [comment]
Thanks, @paulkerschen  I made amends. And yes, CSS linter dinged me.

**Pull request merged**